public/interviews.js

```diff
@@ -344,3 +351,31 @@ As always, a huge *Thank You* to the band for their time, openness and availabil
     liked: false,
   },
 ];
+
+// SEARCH FUNCTION TEST
+
+export default function SearchResults({ searchTerm }) {
+  const results = interviews.filter((interview) =>
+    interview.tags.some((tag) => tag.includes(searchTerm))
+  );
+
+  if (!searchTerm || !results.length) {
+    return <p>No results</p>;
+  }
+
+  return (
+    <ul>
+      {results.map((interview) => (
+        <li key={interview.id}>
+          <h3>{interview.title}</h3>
+          <p>{interview.text}</p>
+          <ul>
+            {interview.tags.map((tag) => (
+              <li key={tag}>{tag}</li>
+            ))}
+          </ul>
+        </li>
+      ))}
+    </ul>
+  );
+}
```

Comment:
Probably doesn't need to be part of the PR. 👀 ?

Reply:
I most likely created a snafu and will have to pay more attention XD